### PR TITLE
README

### PR DESCRIPTION
**Pull request by ImIzGuderDanU**
i put general description of our team

# what Would you like to do 

Why Charlie

### CODE REVIEW

**Comment by BaconBoi95** [comment]
# Welcome
This is the readme thread. Once everybody likes the readme. WE WILL START THE PULL

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Epic Pi Bots
+We are the elkins pointe middle school robotics team,
+and this is our website (Mainframe)!
+This site will have updates on what were doing.
+# EPMS vs Highschool and other middle schools.
```
Comment (End3rYT) [inline readme.md]: Remember

#- Makes Heading 1
##- Makes Heading 2

**Comment by BaconBoi95** [comment]
We need to reset the pull
uhh just gimme a sec

**Comment by BaconBoi95** [comment]
Go to #3

**Pull request closed without merging**